src/platforms/python/common/metrics/index.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: Set Up Metrics
+description: "Learn how to measure the data points you care about by configuring Metrics in your Python app."
+---
+
+<Note>
+
+Metrics for Python are supported with Sentry Python SDK version `1.38.0` and above.
```

Comment:
Let's put `1.40.0` here to simplify it. It gets confusing pointing out various versions on the same page.